test/acceptance/features/bindAppToService.feature

```diff
@@ -305,3 +304,42 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-a-d-c" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-a-d-c" should be changed to "True"
         And Secret "binding-request-a-d-c" contains "SOME_KEY" key with value "SOME_VALUE:5432:db-demo-a-d-c"
+
+    Scenario: Service Binding Request to fetch data from scalar, sequence, map data structures from Service CR
+	Given Imported Nodejs application "nodejs-rest-http-crud-custom-env-var" is running
+	* DB "db-demo-custom-env-var" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-custom-env-var
+            spec:
+                applicationSelector:
+                    resourceRef: nodejs-rest-http-crud-custom-env-var
+                    group: apps
+                    version: v1
+                    resource: deployments
+                backingServiceSelectors:
+                   - group: postgresql.baiju.dev
+                     version: v1alpha1
+                     kind: Database
+                     resourceRef: db-demo-custom-env-var
+                     id: postgresDB
+                customEnvVar:
+                   - name: TAGS
+                     value: '{{ .postgresDB.spec.tags }}'
+                   - name: ARCHIVE_USERLABEL
+                     value: '{{ .postgresDB.spec.userLabels.archive }}'
+                   - name: SECONDARY_SECRETNAME
+                     value: '{{ .postgresDB.spec.secretName }}'
+            """
+        Then application should be re-deployed
+	And application should be connected to the DB "db-demo-custom-env-var"
+	And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-custom-env-var" should be changed to "True"
+	And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-custom-env-var" should be changed to "True"
+	And Secret "binding-request-custom-env-var" contains "TAGS" key with value "[centos7-12.3 123]"
+	And Secret "binding-request-custom-env-var" contains "ARCHIVE_USERLABEL" key with value "false"
+	And Secret "binding-request-custom-env-var" contains "SECONDARY_SECRETNAME" key with value "[map[primarySecretName:example-primaryuser secondarySecretName:example-secondaryuser] map[rootSecretName:example-secretuser]]"
```

Comment:
We should check the values of injected env variables, similar to how we do that in [the documented example](https://github.com/redhat-developer/service-binding-operator/blob/master/examples/java_postgresql_customvar/README.md#check-the-environment-variable-creation-in-the-application-pod).